client/components/HomePage.jsx

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import { render } from 'react-dom';
+import App from './App.jsx';
+
+class HomePage extends Component {
```

Comment:
I think this would make more sense as a functional component. 
You should be able to remove { render } and App. Components only need to be imported into components that will render them.